provisioning/e2e/_transport_integration.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var fs = require('fs');
+var assert = require('chai').assert;
+var Http = require('../transport/http').Http;
+// var Mqtt = require('../transport/mqtt').Mqtt;
+
+var transportConfig = {
+  idScope: 'scope',
+  userAgent: 'agent'
+};
+var registrationId = '__';
+var x509 = {};
+var registrationBody = {
+
+};
+
+var idScope = process.env.IOTHUB_PROVISIONING_INTEGRATION_IDSCOPE;
+var registrationId = process.env.IOTHUB_PROVISIONING_INTEGRATION_REGISTRATION_ID;
+var x509Key = fs.readFileSync(process.env.IOTHUB_PROVISIONING_INTEGRATION_X509_KEY, 'utf-8');
+var x509Cert = fs.readFileSync(process.env.IOTHUB_PROVISIONING_INTEGRATION_X509_CERT, 'utf-8');
```

Comment:
should dynamically provision those instead of relying of env variable unless absolutely necessary. the road to hell is paved with env variables.

Reply:
Absolutely.  I just didn't want to take a dependency on the service SDK yet and this was the shortest path to get some validation going.

---
In reply to: [146405917](https://github.com/Azure/azure-iot-sdk-node/pull/115#discussion_r146405917) [](ancestors = 146405917)